### PR TITLE
Update BBCode in RichTextLabel documentation for Godot 4.2

### PR DESCRIPTION
**Pull request by Calinou**
- Add `lang` tag.
- Add `tab_stops` parameter to `p`.
- Add `pulse` built-in effect and remove custom pulse effect as it's superseded.

___

- See https://github.com/godotengine/godot/pull/77117 and https://github.com/godotengine/godot/pull/80848.

### CODE REVIEW

File: tutorials/ui/bbcode_in_richtextlabel.rst [hunk 1]
```diff
@@ -352,6 +353,14 @@ Reference
       | ``{text}``
       | ``[/opentype_features]``
 
+  * - | **lang**
+      | Overrides the language for ``{text}`` that is set by the **BiDi > Language** property in RichTextLabel.
+        ``{code}`` must be an ISO :ref:`language code <doc_locales>`.
+        This can be used to enforce the use of a specific script for a language without starting a new paragraph.
+        Some font files may contain script-specific substitutes, in which case they will be used.
```
Comment (YuriSizov) [inline tutorials/ui/bbcode_in_richtextlabel.rst:360]: You should probably wrap lines to the same width as the rest of the table uses (same above, here it's just heavier).

File: tutorials/ui/bbcode_in_richtextlabel.rst [hunk 2]
```diff
@@ -352,6 +353,14 @@ Reference
       | ``{text}``
       | ``[/opentype_features]``
 
+  * - | **lang**
+      | Overrides the language for ``{text}`` that is set by the **BiDi > Language** property in RichTextLabel.
```
Comment (YuriSizov) [inline tutorials/ui/bbcode_in_richtextlabel.rst:357]:
```suggestion
      | Overrides the language for ``{text}`` that is set by the **BiDi > Language** property in ``RichTextLabel``.
```

File: tutorials/ui/bbcode_in_richtextlabel.rst [hunk 3]
```diff
@@ -492,8 +501,17 @@ Paragraph options
   | `Default` | Inherit                                    |
   +-----------+--------------------------------------------+
 
-  Locale override.
+  Locale override. Some font files may contain script-specific substitutes, in which case they will be used.
+
+- **tab_stops**
+
+  +-----------+------------------------------------------------+
+  | `Values`  | List of floating-point numbers, e.g. ``10,30`` |
```
Comment (YuriSizov) [inline tutorials/ui/bbcode_in_richtextlabel.rst:509]: Maybe it's worth showing a floating point number here, so people aren't confused by the comma (it is used as a decimal separator in some languages/locales).

File: tutorials/ui/bbcode_in_richtextlabel.rst [hunk 4]
```diff
@@ -492,8 +501,17 @@ Paragraph options
...
+  +-----------+------------------------------------------------+
+  | `Default` | Width of the space character in the font       |
+  +-----------+------------------------------------------------+
 
+  Overrides the horizontal offsets for each tab character. When the end of the list is reached, the tab stops will loop over. For example, if you set ``tab_stops`` to ``10,30``, the first tab will be at ``10`` pixels, the second tab will be at ``10 + 30 = 40`` pixels, and the third tab will be at ``10 + 30 + 10 = 50`` pixels from the origin of the RichTextLabel.
```
Comment (YuriSizov) [inline tutorials/ui/bbcode_in_richtextlabel.rst:514]: Is it not wrappable?

File: tutorials/ui/bbcode_in_richtextlabel.rst [hunk 5]
```diff
@@ -733,6 +751,21 @@ All examples below mention the default values for options in the listed tag form
...
     line using the effect.
 
+Pulse
+~~~~~
+
+.. image:: img/bbcode_in_richtextlabel_effect_pulse.webp
+
+Pulse creates an animated pulsing effect that multiplies each character's
+opacity and color. It can be used to bring attention to specific text. Its tag
+format is ``[pulse freq=1.0 color=#ffffff40 ease=-2.0]{text}[/fade]``.
```
Comment (YuriSizov) [inline tutorials/ui/bbcode_in_richtextlabel.rst:761]: Tag mismatch :)

File: tutorials/ui/bbcode_in_richtextlabel.rst (hunk 4 above)
Comment (Calinou) [reply tutorials/ui/bbcode_in_richtextlabel.rst:514]: It is, but I forgot to wrap it.

File: tutorials/ui/bbcode_in_richtextlabel.rst (hunk 5 above)
Comment (Calinou) [reply tutorials/ui/bbcode_in_richtextlabel.rst:761]: Good catch! Fixed :slightly_smiling_face:

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Pull request merged**